politeiawww/politeiawww_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+
+	"github.com/decred/politeia/politeiad/api/v1/identity"
+	www "github.com/decred/politeia/politeiawww/api/www/v1"
+)
+
+func TestHandleVersion(t *testing.T) {
+	p, cleanup := newTestPoliteiawww(t)
+	defer cleanup()
+
+	id, err := identity.New()
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+
+	p.cfg.Identity = &id.Public
+
+	expectedResponse := www.VersionReply{
+		Version: www.PoliteiaWWWAPIVersion,
+		Route:   www.PoliteiaWWWAPIRoute,
+		PubKey:  hex.EncodeToString(p.cfg.Identity.Key[:]),
+		TestNet: p.cfg.TestNet,
+	}
+
+	var tests = []struct {
+		name         string
+		wantResponse interface{}
```

Comment:
Why is this an interface?  Why are you never actually using this in the test?

Reply:
Did a confusion with the replies that were meant to be tested. Corrected this and the nomenclature from response to reply.